MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +222,183 @@ this.Content = chart;
 
 {% endtabs %}
 
+### Trackball Label Template
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+        <DataTemplate x:Key="trackballLabelTemplate">
```

Comment:
@EzhilarasanElangovan31 update template from SB, use person image instead of Path

Reply:
Modified